### PR TITLE
[added] Very Important Changes!™ 

### PR DESCRIPTION
**Pull request by mugg91**
## Description

This PR adds some "long overdue" features to the game.
I will detail each new feature below.

### New Arrow for Archer

Well, it's not exactly new, but I replaced the normal arrow because everyone has been complaining lately about arrows.
Arrows cost too much. Arrows fill up the battleground too much. Arrows this. Arrows that.
So I came up with an interesting new Arrow that fits the Archer character perfectly, will increase the depth of the game and gain everyone's attention.
With new sound effects.

arrow

### Better Drill

I'm really a fan of the Anime "Gurren Lagann", where they battle against all odds in gigantic robo-mechas with those huge drills for hands.
And look, King Arthur's Gold even has a Kamina head. The devs must be fans of that show, too.
But the Drill they made is super mediocre. It can only destroy like 2 tiles before Builder has to drop it because of what? Burning?
What a joke. So I buffed it just a very tiny bit. You won't even notice!
"MY DRILL IS THE DRILL THAT WILL PIERCE THE HEAVENS!!"
Kamina would be proud if you add this change to the game.

drill

### Better Dinghy

I always liked Dinghy because you could buy it on a whim and bring it to the battle for trollin- ... I mean, for doing very clever plays, such as blocking the enemy doors and blocking the enemy players, too.
However, it felt like it didn't serve its purpose well enough. It's also very small, like only a few pixels.
So in order to make it better at being a trolling dev- ... I mean, a strategic device, I made it a bit bigger. Just by a few pixels.
Also you can bring more friends to ride on it. Win for everyone!

dinghy1

### Improved TDM

To be more in line with what the TDM game mode is supposed to be, I made a few inevitable changes.
Since TDM stands for "Team Dance Match" I gave TDM_Ruins a cheerier look.
In fact we should rename it to "Birthday Mode". I even added some balloons.
Also Trader, despite gaining all that money from those noob KAG players, looked like he had one foot in the grave so I improved him.
He used the money to buy himself a nice black suit and a trumpet.
From now on, he will show off his musical skills to you.

ruins

### New Sprite for Mine

What people sometimes forget is that King Arthur's Gold is actually an educational game. I like it because it really captures what medieval combat is all about and pays close attention to details.
From playing this game, I learned that back then there used to be suicidal Knights that flew in the sky and hillbillies that drilled through the entire landscape.
But one thing still seemed out of place to me: The Mine. A spiky bomb that just explodes when going too close. That seems slightly unrealistic to me.
So I had an idea to improve the game further and to be more realistic.
Since Knights lacked toilets back then, they often had to go in the bushes and do their "business" there.
So in other words, there was a lot of poop. And now, the Mine item will be a poop item.
It fits perfectly, because of course, you wouldn't want to step on a poop, would you?

zzzz

### New Tent

I always wondered what was up with that bird duck thing on the Tent in CTF.
It just never moved. Really odd, because birds are supposed to be living animals.
So I changed tent to make it more alive.
You're welcome.

duck

### CODE REVIEW

**Comment by GELGELAT** [comment]
WE NEED A NEW EVENT FOR APRIL 1ST!!!!!!!!!!!! Mugg you are god of code!

**Comment by mugg91** [comment]
This PR has lived its life.

As a note, any code from this PR is free to use in anyone's project or in any upcoming april joke build.

I'm most fond of the balloons particles, perhaps that can be re-used in some way, maybe in the birthday event.
